addons/stock/views/stock_move_line_views.xml

```diff
@@ -86,9 +86,9 @@
                         </group>
                         <group>
                             <label for="quantity" string="Quantity"/>
-                            <div class="o_row">
-                                <field name="quantity"/>
-                                <field name="product_uom_id" options="{'no_create': True}" string="Unit of Measure" groups="uom.group_uom"/>
+                            <div class="row">
+                                <div class="col-3"><field name="quantity"/></div>
+                                <div class="col-3"><field name="product_uom_id" options="{'no_create': True}" string="Unit of Measure" groups="uom.group_uom"/></div>
```

Comment:
Why do you use sometime `col-2` and sometime `col-3` ?
Also, don't forget the view can also be displayed through a mobile device and so a small screen.
Following suggestion is to handle mobile screen
```suggestion
                                <div class="col col-md-3"><field name="quantity"/></div>
                                <div class="col"><field name="product_uom_id" options="{'no_create': True}" string="Unit of Measure" groups="uom.group_uom"/></div>
```
I didn't check every views but I guess the issue of having too much small fields on mobile should be addressed for other modified views too